src/config/dynamic-theme-fixes.config

```diff
@@ -19441,6 +19441,19 @@ INVERT
 
 ================================
 
+s.weibo.com
+
+INVERT
+#woo_svg_nav_logo > g > path:nth-child(5)
```

Comment:
Something feels wrong about this selector, it target's a 5th path element in a logo. BUt I hope you know what you are doing.

Reply:
![image](https://github.com/darkreader/darkreader/assets/24503369/2d4ad86c-ec55-487a-8289-ac7761d7c867)
original fill is covered by a light color, it makes the logo center invisible, so i INVERT that part.😋
![image](https://github.com/darkreader/darkreader/assets/24503369/b69dd61e-893b-4462-9929-0a145c4f126b)